Code/Framework/AzToolsFramework/Tests/FocusMode/EditorFocusModeTests.cpp

```diff
@@ -6,123 +6,90 @@
  *
  */
 
-#include <AzTest/AzTest.h>
-#include <AzCore/UserSettings/UserSettingsComponent.h>
-#include <AzCore/Component/TransformBus.h>
-#include <AzToolsFramework/FocusMode/FocusModeInterface.h>
-#include <AzToolsFramework/UnitTest/AzToolsFrameworkTestHelpers.h>
+#include <Tests/FocusMode/EditorFocusModeFixture.h>
 
 namespace AzToolsFramework
 {
-    class EditorFocusModeTests
-        : public ::testing::Test
+    TEST_F(EditorFocusModeFixture, EditorFocusModeTests_SetFocus)
```

Comment:
suggestion: I'm being a bit pedantic here but in a perfect world we'd have `EditorFocusModeSetFocusFocussesEntity` and `EditorFocusModeClearFocusClearsFocussedEntity` or something similar (we're still testing two things in this test 😉)

Reply:
Done :D